BlazarService/src/test/java/com/hubspot/blazar/service/DependencyVersionTest.java

```diff
@@ -43,11 +45,10 @@
   @Inject
   private DependenciesService dependenciesService;
 
+  @Inject
```

Comment:
You don't need that Inject annotation. Jukito will auto inject the arguments of your test methods without that.